_data/simple-icons.json

```
@@ -1291,6 +1291,11 @@
             "hex": "F9DC3E",
             "source": "https://github.com/babel/website/blob/93330158b6ecca1ab88d3be8dbf661f5c2da6c76/website/static/img/babel-black.svg"
         },
+        {
+            "title": "Backblaze",
+            "hex": "D30035",
+            "source": "https://upload.wikimedia.org/wikipedia/commons/9/9b/Backblaze_Logo.svg"
```

```suggestion
            "source": "https://www.backblaze.com/company/news.html"
```

Their press kit can be found here.